src/components/StablecoinAccordion/use-stablecoin-accordion.ts

```diff
@@ -0,0 +1,311 @@
+import { useStaticQuery, graphql } from "gatsby"
```

Comment:
For consistency, lets keep using camelCase names. This file should be named `useStablecoinAccordion.ts`

Reply:
You got it!

The [Trilemma component](https://github.com/ethereum/ethereum-org-website/tree/dev/src/components/Trilemma) would need an update with its similar file.

Reply:
Oh did't see that one. Good catch, yea. Will do a quick refactor.